lib/pages/timetable/service/google_calendar_services.dart

```diff
@@ -0,0 +1,138 @@
+import 'package:acs_upb_mobile/pages/classes/model/class.dart';
+import 'package:acs_upb_mobile/pages/timetable/model/events/recurring_event.dart';
+import 'package:acs_upb_mobile/pages/timetable/model/events/uni_event.dart';
+import 'package:acs_upb_mobile/pages/timetable/service/uni_event_provider.dart';
+import 'package:acs_upb_mobile/resources/google_apis.dart';
+import 'package:flutter_web_browser/flutter_web_browser.dart';
+import 'package:googleapis/calendar/v3.dart' as g_cal;
+import 'package:googleapis_auth/auth_io.dart';
+import 'package:acs_upb_mobile/resources/utils.dart';
+
+extension GoogleCalendarServices on UniEventProvider {
+  g_cal.Event convertEvent(UniEvent uniEvent) {
+    final g_cal.Event googleCalendarEvent = g_cal.Event();
+
+    final g_cal.EventDateTime start = g_cal.EventDateTime();
+    final DateTime startDateTime = uniEvent.start.toDateTimeLocal();
+
+    start
+      ..timeZone = 'Europe/Bucharest'
+      // google calendar has different timezone formats
```

Comment:
Different from what?
```suggestion
      // Google Calendar uses the IANA timezone format, but the native Dart `DateTime` uses an abbreviation provided by the operating system.
```